src/SFA.DAS.EmployerIncentives.Domain.UnitTests/ValueObjects/WhenCreatingTheIncentive.cs

```diff
@@ -11,25 +11,35 @@ namespace SFA.DAS.EmployerIncentives.Domain.UnitTests.ValueObjects
     [TestFixture]
     public class WhenCreatingTheIncentive
     {
-        private List<IncentivePaymentProfile> _incentivePaymentProfiles;
+        private IncentivesConfiguration _incentivePaymentProfiles;
 
         [SetUp]
         public void SetUp()
         {
-            _incentivePaymentProfiles = new List<IncentivePaymentProfile>
+            var profiles = new List<IncentivePaymentProfile>
             {
-                new IncentivePaymentProfile(IncentiveType.TwentyFiveOrOverIncentive,
-                    new List<PaymentProfile>
-                        {new PaymentProfile(90, 1000), new PaymentProfile(365, 1000)}),
+                new IncentivePaymentProfile(
```

Comment:
The Phase should be a value object sine that is what is stored against the application and incentive.  And created as new Phase(string version);

Then the IIncentivePaymentProfileService can be extended with

IncentivePaymentProfile Get(Phase phase)

to return the IncentivePaymentProfile data for the phase.